sql/moz-fx-data-shared-prod/mozilla_vpn_derived/all_subscriptions_v1/query.sql

```diff
@@ -232,6 +234,7 @@ apple_iap_subscriptions AS (
     created_at AS created,
     CAST(NULL AS TIMESTAMP) AS trial_end,
     CAST(NULL AS TIMESTAMP) AS canceled_at,
+    CAST(NULL AS STRING) AS cancelled_for_customer_at,
```

Comment:
Should this be called `canceled_for_customer_at`?

```suggestion
    CAST(NULL AS STRING) AS canceled_for_customer_at,
```

Reply:
yes, tyty